src/client.js

```diff
@@ -3991,14 +3993,25 @@ MatrixClient.prototype.getCanResetTimelineCallback = function() {
  */
 MatrixClient.prototype.relations =
 async function(roomId, eventId, relationType, eventType, opts = {}) {
+    const isEncrypted = this.isRoomEncrypted(roomId);
+    const fetchedEventType = isEncrypted ? "m.room.encrypted" : eventType;
```

Comment:
Hmm, so this `fetchedEventType` will be wrong for reactions, which are now always `m.reaction` even in E2E rooms. 😓 It does seem nice to handle encryption transparently though... Maybe we should add a helper function similar to `_encryptEventIfNeeded` called something like `_getEncryptedIfNeededEventType` that returns the eventual event type (with a similar exception for reactions)?